app/src/main/java/com/nutomic/syncthingandroid/util/ConfigXml.java

```diff
@@ -156,17 +245,37 @@ public void updateIfNeeded() {
             changed = true;
         }
 
+        /* Section - options */
         // Disable weak hash benchmark for faster startup.
         // https://github.com/syncthing/syncthing/issues/4348
         Element options = (Element) mConfig.getDocumentElement()
                 .getElementsByTagName("options").item(0);
         changed = setConfigElement(options, "weakHashSelectionMethod", "never") || changed;
 
+        /* Check if we have to dismiss any specific "unackedNotificationID" */
+        /* Dismiss "fsWatcherNotification" according to https://github.com/syncthing/syncthing-android/pull/1051 */
+        if (getConfigElement(options, "unackedNotificationID").contains("fsWatcherNotification")) {
+          Log.i(TAG, "Remove option 'unackedNotificationID' to dismiss 'fsWatcherNotification'.");
+          // According to review, it is sufficient to remove all "unackedNotificationID" contents.
+          // changed = setConfigElement(options, "unackedNotificationID", getConfigElement(options, "unackedNotificationID").replace("fsWatcherNotification", "")) || changed;
+          options.removeChild(options.getElementsByTagName("unackedNotificationID").item(0));
+          changed = true;
+        }
```

Comment:
This is not handled correctly, there can be multiple `unackedNotificationID` elements, each with a different value.
You check it seems only the first one, and remove only the first one.

Reply:
Ok, switched back to my first version only removing the string "fsWatcherNotification" from the unackedNotificationID node.